packages/shared/src/components/TagLinks.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import Link from 'next/link';
+import { Button } from './buttons/Button';
+
+interface TagLinkProps {
+  tag: string;
+  className?: string;
+}
+
+export function TagLink({ tag, className }: TagLinkProps): ReactElement {
+  return (
+    <Link href={`/tags/${tag}`} passHref key={tag}>
```

Comment:
I'm not sure if this should really be a link. You might want to ask it on the #product channel

Reply:
Yes, [Tsahi has confirmed](https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/26280164/Profile+Tooltip?focusedCommentId=112394253#comment-112394253) that this should be a link to allow users to navigate to that specific tag.